README.md

```diff
@@ -1 +1,12 @@
-# encode_repeating
\ No newline at end of file
+# Encode Repeating
```

Comment:
Should we link them to the [Wikipedia article for run-length encoding](https://en.wikipedia.org/wiki/Run-length_encoding) for more context around this idea?